project_issue_task_sync/models/project_issue.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ProjectIssue(models.Model):
+    _inherit = 'project.issue'
+
+
+    def get_changed_vals(self, task):
+        vals = {}
+        if len(task.issue_ids) ==1:
```

Comment:
and if there are multiple issues? use falsyness here and deal with multiple issues below. per assumption all stages should be the same anyways, so it's safe to use the first issue